### PR TITLE
Feat/gwas v2 cleanup

### PR DESCRIPTION
**Pull request by jarvisraymond-uchicago**
Jira Ticket: [VADC-437](https://ctds-planx.atlassian.net/browse/VADC-437)

### New Features
This updates the code base associated with the GWASApp and GWASResults application by cleaning up unneeded CSS styles and changing the names of files and selectors from ```GWASV2``` to ```GWASApp```. 

This update:
- Limits GWASResults stylesheet to only style the GWASResult application and moves necessary styles into ```src/Analysis/GWASV2/GWASV2.css``` from ```GWASUIApp.css```.
- Renames the file ```GWASUIApp.css``` to ```GWASResults.css```
- Removes any references in ```src/Analysis/GWASV2``` to ```src/Analysis/GWASResults/GWASUIApp.css```. 
- Renames ```src/Analysis/GWASV2``` to ```src/Analysis/GWASApp```
- Renames all other instances of GWASV2 to GWASApp in file names and selectors within the GWASApp

Following this update, the visual appearance and functionality of both applications should remain unchanged. 

[VADC-437]: https://ctds-planx.atlassian.net/browse/VADC-437?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Analysis/GWASResults/GWASResults.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.ant-btn {
+  width: auto;
+}
+
+.GWASApp-jobStatus {
```
Comment (pieterlukasse) [inline src/Analysis/GWASResults/GWASResults.css:5]: maybe call it `GWASResults-jobStatus`?

File: src/Analysis/GWASResults/GWASResults.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+.GWASApp-jobStatus .ant-collapse {
+  width: 100%;
+}
+
+.ant-list-items {
+  content: center;
+}
+
+.GWASUI-flexRow {
```
Comment (pieterlukasse) [inline src/Analysis/GWASResults/GWASResults.css:35]: maybe call it `GWASResults-flexRox`?

File: src/Analysis/GWASApp/GWASApp.css [hunk 3]
```diff
@@ -0,0 +1,260 @@
+.GWASApp .steps-content {
```
Comment (pieterlukasse) [inline src/Analysis/GWASApp/GWASApp.css:1]: When do we call something `GWASApp` vs `GWASUI`?

**Review by pieterlukasse — COMMENTED** [review]
Looks good to me!
Just a few minor comments/questions regarding naming in .css

File: src/Analysis/GWASResults/GWASResults.css (hunk 1 above)
Comment (jarvisraymond-uchicago) [reply src/Analysis/GWASResults/GWASResults.css]: Updated.

File: src/Analysis/GWASApp/GWASApp.css (hunk 3 above)
Comment (jarvisraymond-uchicago) [reply src/Analysis/GWASApp/GWASApp.css:1]: ```GWASUI``` is a class name ported over from the initial version of the stylesheet: when we created the new version the app, we copied the style sheet  and much of the markup associated with the first version of the application. 

GWASApp is a class name introduced per your suggestion for this ticket: all instances of the class name ```GWASV2``` have been changed to ```GWASApp```

**Review by pieterlukasse — APPROVED** [review]
thanks @jarvisraymond-uchicago !

**Pull request merged**